### PR TITLE
Update the README file

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,25 +4,78 @@
 
 # DevExpress Reporting for .NET MAUI - Generate Bound-Data Report
```
Comment (Abadzhev) [inline README.md]:
```suggestion
# DevExpress Reporting for .NET MAUI - Generate and Display a Data-Bound Report
```

File: README.md [hunk 2]
```diff
@@ -4,25 +4,78 @@
...
 
-...
+This example generates a data-bound report in the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and displays it in the [PDF Viewer for .NET MAUI](https://docs.devexpress.com/MAUI/404632/pdf-viewer/pdf-viewer). To use Reporting APIs in a .NET MAUI project, you need a license that includes [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/).
```
Comment (Abadzhev) [inline README.md]: In the description, you skip the part that actually creates the report layout. The first step already creates an `XtraReportInstance` object... But that object is a report with predefined layout. And to understand what this report does, our customers would need to review files in the **ReportLibrary** folder. 

We can go two different ways: 
- Mention the ReportLibrary folder and say that they can review those files and that this is where `XtraReportInstance` is defined. 
- Describe the entire process of how to create a report. 

I'm guessing you don't want to describe how to create a report, so we'll go with the first option. 

```suggestion
This example uses [DevExpress PDF Viewer for .NET MAUI](https://docs.devexpress.com/MAUI/404632/pdf-viewer/pdf-viewer) to display a data-bound report in a mobile app. 

You can adjust this example to use any previously-defined report. In this repository, the **ReportLibrary** folder contains the data source and the report layout definition. We constructed the report with the help of the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and named the report class `XtraReportInstance`. The code below creates this object to generate a report document. 

**NOTE**: To use Reporting APIs in a .NET MAUI project, you need a license that includes [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/).
```

File: README.md [hunk 3]
```diff
@@ -4,25 +4,78 @@
...
+        await Task.Run(() =>
+        {
+            report = new XtraReportInstance() { Name = "Sample" };
+            report.CreateDocument();
+        });
+        return report;
+    }
+    ```
+
+2. Call the `CreateTableReportAsync()` function to generate a new report and get a path to the report:
```
Comment (Abadzhev) [inline README.md]: The path is just the file where you will export the report. You use the same line in the next step - no need to mention it here. 

```suggestion
2. When the page loads, call the `CreateTableReportAsync()` function to generate a new report:
```

File: README.md [hunk 4]
```diff
@@ -4,25 +4,78 @@
...
+
+    ```csharp
+    private async void ContentPage_Loaded(object sender, EventArgs e) {
+        XtraReport report = await CreateTableReportAsync();
+        string resultFile = Path.Combine(FileSystem.Current.AppDataDirectory, report.Name + ".pdf");
```
Comment (Abadzhev) [inline README.md]: No need for this line - you have it in the next step (and it makes more sense there).

```suggestion

```

File: README.md [hunk 5]
```diff
@@ -4,25 +4,78 @@
...
+        // ...
+    }
+    ```
+
+3. Call the `ExportToPdf(resultFile)` method to convert the report to PDF:
+
+    ```csharp
+    private async void ContentPage_Loaded(object sender, EventArgs e) {
+        XtraReport report = await CreateTableReportAsync();
```
Comment (Abadzhev) [inline README.md]: I wouldn't repeat the same lines in every code snippet: 

```suggestion
        // ...
```

File: README.md [hunk 6]
```diff
@@ -4,25 +4,78 @@
...
+    }
+    ```
+
+4. Use the `PdfViewer.DocumentSource` property to open the report in the PDF Viewer control:
+
+    ```csharp
+    private async void ContentPage_Loaded(object sender, EventArgs e) {
+        XtraReport report = await CreateTableReportAsync();
+        string resultFile = Path.Combine(FileSystem.Current.AppDataDirectory, report.Name + ".pdf");
+        report.ExportToPdf(resultFile);
```
Comment (Abadzhev) [inline README.md]: Again, no need to repeat the same lines: 

```suggestion
        // ...
```

File: README.md [hunk 7]
```diff
@@ -2,27 +2,81 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# DevExpress Reporting for .NET MAUI - Generate Bound-Data Report
+# DevExpress Reporting for .NET MAUI - Generate and Display a Data-Bound Report
 
-...
+This example uses [DevExpress PDF Viewer for .NET MAUI](https://docs.devexpress.com/MAUI/404632/pdf-viewer/pdf-viewer) to display a data-bound report in a mobile app. 
+
+You can adjust this example to use any previously-defined report. In this repository, the **ReportLibrary** folder contains the data source and the report layout definition. We constructed the report with the help of the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and named the report class `XtraReportInstance`. The code below creates this object to generate a report document.
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
You can adjust this example to use any existing report. In this repository, the **ReportLibrary** folder contains the data source and the report layout definition. We constructed the report with the help of the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and named the report class `XtraReportInstance`. The code below creates this object to generate a report document. 
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**